definition/ILivechatVoipServiceResult.ts

```diff
@@ -0,0 +1,9 @@
+export interface IAgentExtensionMap {
```

Comment:
Can you rename this file to `IOmnichannelVoipServiceResult`?

Reply:
Fixed.